File: practice20/5.cpp
//https://www.nowcoder.com/practice/dd63c30dfef04770b5813e63f5a2615a?tpId=98&tqId=32881&tPage=3&rp=3&ru=/ta/2019test&qru=/ta/2019test/question-ranking
#include<iostream>
#include<string>
using namespace std;

int main()
{
	string str;
	while (cin >> str)
	{
		int m = str.size();
		if (str[0] == '-')
		{
			cout << '-';
			for (int i = m - 1; i > 0; i--)
			{
				cout << str[i];
			}
			cout << endl;
		}
		else
		{
			for (int i = m - 1; i >= 0; i--)
			{
				cout << str[i];
			}
			cout << endl;
		}
	}
	system("pause");
	return 0;
}

File: practice28/5.java


//https://www.nowcoder.com/practice/1664fe871878496aa600b6e09557982b?tpId=98&tqId=33045&tPage=12&rp=12&ru=/ta/2019test&qru=/ta/2019test/question-ranking
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner scanner = new Scanner(System.in);
        String s = scanner.nextLine();
        String[] str = s.split(" ");
        int[] pt = new int[str.length];
        for (int i = 0; i < str.length; i++) {
            pt[i] = Integer.parseInt(str[i]);
        }
        Arrays.sort(pt);
        for (int i = 0; i < str.length - 1; i++) {
            if (pt[i] == pt[i + 1]) {
                System.out.println(pt[i]);
                break;
            }
        }
    }
}